### PR TITLE
Add Lit icon and fix typo in the Contributions guidelines

### PR DESCRIPTION
**Pull request by dzintars**
lit

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
  - Color picked as most prominent logo color
  - https://github.com/lit/lit/ (7.5k stars)
  - https://lit.dev/

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -4922,6 +4922,11 @@
             "hex": "ED3237",
             "source": "https://lionairthai.com/en/"
         },
+        {
+            "title": "Lit",
+            "hex": "324FFF",
+            "source": "https://lit.dev/images/flame-favicon.svg"
```
Comment (adamrusted) [inline _data/simple-icons.json:4928]: Could you update the source for this to https://github.com/lit/lit.dev/blob/5e59bdb00b7a261d6fdcd6a4ae529e17f6146ed3/packages/lit-dev-content/site/images/flame-favicon.svg - as that's the source-code of the website. Should allow us better tracking should they change / remove this file in the future!

**Review by adamrusted — CHANGES_REQUESTED** [review]
Thanks for submitting this one @dzintars, and welcome to Simple Icons!
The icon looks spot on from my perspective, apart from the [apparent lines throughout the shapes](https://user-images.githubusercontent.com/1830867/116809752-9538a300-ab37-11eb-8ef9-bdffebc88fca.png) which aren't present in the source SVG. Any chance you could look at removing them, or rebuilding? I've also commented below on the source URL for the icon. Just to give us a bit more of a paper-trail for the SVG.

Good spot on the `CONTRIBUTING.md` typo! 🧐

File: _data/simple-icons.json (hunk 1 above)
Comment (dzintars) [reply _data/simple-icons.json]: Thank you for your time. Learned something new. :) I have several additional icons incoming, so this review is valuable to me.
I changed the source and jumped into Illustrator to compound those paths. I was something related to my tooling. Now the icon should be even smaller. Linting and tests pass.

**Review by adamrusted — APPROVED** [review]
LGTM! Thanks for the update @dzintars! 🚀

**Pull request merged**